docs/features.md

```diff
@@ -98,3 +98,10 @@ The feature defaults a service account's `automountServiceAccountToken` to false
 |7| **not defined** | false | false | false |
 |8| true | not defined | true | true |
 |9| false | not defined | false | false |
+
+## Karydia Exclusion Handling
+
+Namespaces and other objects can be opted out of being "watched" by karydia. Therefor, there are two options:
+- (nearly) each feature provides its own annotation for namespaces and / or other objects to be ignored by the respective karydia feature - see feature descriptions above
```

Comment:
Would not add a space between "and / or".

Reply:
done